devServer v4: options has an unknown property 'before'

milosivanovic opened the pull request:
Fixes #83

myufa commented:
https://stackoverflow.com/questions/66061020/when-i-update-to-webpack5-there-is-a-error-configuration-has-an-unknown-proper
Seems like "before" is used in index.js when it should be "onBeforeSetupMiddleware",  hope this helps, awaiting the fix!

milosivanovic commented:
@thedavidprice Thanks for your review! I hope I addressed those comments in my latest commit.

The 404 error is strange, and yes, it does only show when the plugin is used. I can say that the functionality works despite the 404, which puts the plugin in a better place than it is on the main branch (not working at all.) We could either try to resolve that in this PR, or commit a subsequent PR after the issue is identified. I'll leave that up for the maintainer to decide.

@gregberge When you have time to check this out, let us know what you think.

mrnav90 commented:
@gregberge 
Please help check and merge this PR.
We need this update cho webpack 5.

Thanks

iMoses-Apiiro commented:
any updates? we're waiting on this fix otherwise this package is unusable for us

gondar00 commented:
Guys can we please get this merged? @thedavidprice @milosivanovic

thedavidprice commented:
Sorry all, I've no permissions for this repo. And we've now removed the package from RedwoodJS due to the incompatibility so I can't take on responsibility here.

Nudging @gregberge
